### PR TITLE
HPCC-31719 Restore previous values for stub memory resources

### PR DESCRIPTION
**Pull request by ghalliday**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: helm/hpcc/templates/_helpers.tpl [hunk 1]
```diff
@@ -1339,7 +1339,7 @@ Pass in dict with root, me and instances defined
 {{- $stubInstanceResources := .stubResources | default .root.Values.global.stubInstanceResources | default dict }}
 {{- $milliCPUText := $stubInstanceResources.cpu | default "200m" }}
 {{- $milliCPUs := int (include "hpcc.k8sCPUStringToMilliCPU" $milliCPUText) }}
-{{- $memoryText := $stubInstanceResources.memory | default "50Mi" }}
+{{- $memoryText := $stubInstanceResources.memory | default "500Mi" }}
```
Comment (jakesmith) [inline helm/hpcc/templates/_helpers.tpl:1342]: previous value was 200Mi ?

**Review by jakesmith — COMMENTED** [review]
@ghalliday - previous value was 200Mi - are you deliberately upping to 500Mi (maybe because of lack of instance multiplier) ?

File: helm/hpcc/templates/_helpers.tpl (hunk 1 above)
Comment (ghalliday) [reply helm/hpcc/templates/_helpers.tpl]: Just looking at that memory again.  Previously for eclagent it was 200 * max active, same for thor stub.
So actually 800 for eaclagent stub and 400 for thor by default.  Should I just use 400 for all?

**Review by jakesmith — APPROVED** [review]
@ghalliday - looks good.

**Comment by AttilaVamos** [comment]
@ghalliday @jakesmith do you know why the helm chart test failed?

**Comment by jakesmith** [comment]
> @ghalliday @jakesmith do you know why the helm chart test failed?

The helm test intentionally fails if there are any generative changes, to help spot unintentional changes.
In this case - the change is expected.

**Pull request merged**